python/mlc_llm/serve/entrypoints/openai_entrypoints.py

```diff
@@ -428,7 +440,7 @@ async def request_chat_completion(
         async def completion_stream_generator() -> AsyncGenerator[str, None]:
             async_engine.record_event(request_id, event="invoke generate")
             finish_reasons: List[Optional[str]] = [None for _ in range(generation_cfg.n)]
-            async for delta_outputs in async_engine.generate(prompt, generation_cfg, request_id):
+            async for delta_outputs in async_engine.generate(prompt, generation_cfg, request_id):  # type: ignore # pylint: disable=line-too-long
```

Comment:
The original line has 85 characters, which fits our black and pylint limit (that is 100 characters). I assume there is no need to add the ignore and disable pylint here. Is there anything wrong with your settings?

https://github.com/mlc-ai/mlc-llm/blob/edffce44c55539ca43c3eff4b4022dd628205cb7/pyproject.toml#L22-L34

Reply:
I was getting mypy errors, I changed List->Sequence to handle those errors. Removing `# type: ignore # pylint: disable=line-too-long` now